content/en/platform/corda/4.8/enterprise/flow-state-machines.md

```diff
@@ -662,7 +662,7 @@ loaded off disk again.
 If a node has flows still in a suspended state, with flow continuations written to disk, it will not be
 possible to upgrade that node to a new version of Corda or your app, because flows must be completely “drained”
 before an upgrade can be performed, and must reach a finished state for draining to complete (see
-[Draining the node](upgrading-cordapp.md#draining-the-node) for details). While there are mechanisms for “evolving” serialised data held
+[Draining the node](../../../../../en/platform/corda/4.8/enterprise/upgrading-cordapp.html#draining-the-node) for details). While there are mechanisms for “evolving” serialised data held
```

Comment:
Somewhere on this page is an anchor link to #subflows. It should be #sub-flows.

Reply:
This is also making me realize I have always written subflows, not sub-flows.. Uh oh.